src/app/test-record/vehicle/vehicle.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { VehicleComponent } from './vehicle.component';
+import { TestResultModel } from '@app/models/test-result.model';
+import { SharedModule } from '@app/shared/shared.module';
+
+describe('VehicleComponent', () => {
+  let component: VehicleComponent;
+  let fixture: ComponentFixture<VehicleComponent>;
+  const testRecord = {} as TestResultModel;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [SharedModule],
+      declarations: [VehicleComponent]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(VehicleComponent);
+    component = fixture.componentInstance;
+    component.testRecord = testRecord;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
I think we need to add one snapshot for that as well please.

Reply:
Added in 10278